app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -233,7 +247,7 @@ public function getLogoutReasonMessage()
     /**
      * Get sessions for current user
      *
-     * @return \Magento\Security\Model\ResourceModel\AdminSessionInfo\Collection
+     * @return Collection
```

Comment:
Can you please add a full class reference here as we are following the same in Core and it's easy to identify class.